### PR TITLE
Rename "Integrated Console" to "Extension Terminal"

### PR DESCRIPTION
**Pull request by andschwa**
Resolves https://github.com/PowerShell/vscode-powershell/issues/4053.

### CODE REVIEW

**Comment by andschwa** [comment]
@JustinGrote did I miss anything?

**Comment by JustinGrote** [comment]
@andschwa the most visible occurances of it are covered at least, LGTM

File: tools/ReleaseTools.psm1 [hunk 1]
```diff
@@ -82,7 +82,7 @@ function Get-Bullets {
             'Area-Documentation'        = '📖'
             'Area-Engine'               = '🚂'
             'Area-Folding'              = '📚'
-            'Area-Integrated Console'   = '📟'
+            'Area-Extension Terminal'   = '📟'
```
Comment (andschwa) [inline tools/ReleaseTools.psm1:85]: I renamed the labels on GitHub too.

File: src/session.ts [hunk 2]
```diff
@@ -477,7 +481,7 @@ export class SessionManager implements Middleware {
             new PowerShellProcess(
                 this.PowerShellExeDetails.exePath,
                 this.bundledModulesPath,
-                "PowerShell Integrated Console",
+                "PowerShell Extension",
```
Comment (andschwa) [inline src/session.ts:484]: Explicitly no "Terminal" addendum because this is what's displayed in the list of terminals.

File: src/session.ts [hunk 3]
```diff
@@ -220,9 +220,13 @@ export class SessionManager implements Middleware {
         if (this.sessionSettings.integratedConsole.suppressStartupBanner) {
             this.editorServicesArgs += "-StartupBanner '' ";
         } else {
-            const startupBanner = `=====> ${this.HostName} Integrated Console v${this.HostVersion} <=====
+            const startupBanner = `${this.HostName} Extension v${this.HostVersion}
+Copyright (c) Microsoft Corporation.
+
+https://aka.ms/powershell
```
Comment (andschwa) [inline src/session.ts:226]: @SydneyhSmith Is there a better link we could put here?

Comment (SydneyhSmith) [reply src/session.ts:226]: This is what shows on startup? Seems fine to me...alternative would be a VSCode PowerShell link

File: tools/ReleaseTools.psm1 (hunk 1 above)
Comment (SydneyhSmith) [reply tools/ReleaseTools.psm1:85]: You think of everything!

**Review by SydneyhSmith — APPROVED** [review]

**Review by SeeminglyScience — APPROVED** [review]
LGTM!

**Pull request merged**